.github/workflows/master-e2e.yaml

```diff
@@ -225,6 +225,9 @@ jobs:
       RANCHER_UPGRADE: ${{ inputs.rancher_upgrade }}
       UPGRADE: ${{ inputs.upgrade }}
       TIMEOUT_SCALE: 3
+      # K8s Version Upgrade
+      K8S_VERSION_UPGRADE: ${{ inputs.k8s_version_upgrade }}
```

Comment:
Given that this for downstream Kubernetes version upgrade, could this please be renamed to something that specifies that?, for instance `DS_CLUSTER_K8S_VERSION_UPGRADE` OR  `K8S_VERSION_UPGRADE_DS_CLUSTER`?

Reply:
Nice suggestion, I will update it. I was not finding nice name, thanks :)